.github/workflows/docker-ci.yaml

```diff
@@ -59,6 +59,24 @@ jobs:
             ${{ runner.os }}-maven-
       - name: Build docker image
         run: export ES_VERSION=${{ matrix.es }} && export TAG=${{ matrix.es }} && (make docker || make docker)
+      - name: Scan OAP image
+        uses: aquasecurity/trivy-action@master
```

Comment:
Should we lock the version/commit id? Depending on master would be unstable

Reply:
make sense. I will pick up a sha1 later